spanner/spannertest/inmem.go

```diff
@@ -593,7 +593,7 @@ func (s *server) readStream(ctx context.Context, tx *transaction, send func(*spa
 			return err
 		}
 		rsm.RowType.Fields = append(rsm.RowType.Fields, &spannerpb.StructType_Field{
-			Name: ci.Name,
+			Name: string(ci.Name), // TODO: should this be quoting via ci.Name.SQL()?
```

Comment:
I don't think this should be quoted. Row type metadata is returned from the server without quoting, regardless whether the identifier needs quoting when used in SQL or not.

Reply:
Yeah, you're right. This is also potentially not a plain identifier anyway (there's a TODO to do something nicer if the result column is an expression, for instance), but there's no need to quote here.